docker-compose.yaml

```diff
@@ -214,3 +215,20 @@ services:
       OPENSEARCH_PASSWORD: admin
     depends_on:
       - opensearch
+  cypress:
```

Comment:
Please, add this to the `local-config/docker-compose.testrunner.yaml` file, so we keep "production" containers separated from "test" containers. You can use https://github.com/kubeshop/tracetest/blob/main/run-tracetesting-tests.sh as an example